instrumentation/netty/netty-4.1/library/src/main/java/io/opentelemetry/instrumentation/netty/v4_1/NettyClientTelemetryBuilder.java

```diff
@@ -111,8 +112,8 @@ public NettyClientTelemetry build() {
         new NettyClientInstrumenterFactory(
                 openTelemetry,
                 "io.opentelemetry.netty-4.1",
-                false,
-                false,
+                NettyConnectionInstrumentationFlag.DISABLED,
+                NettyConnectionInstrumentationFlag.DISABLED,
```

Comment:
Why does this one not use the config?

Reply:
It's a library instrumentation, I think it just can't implement the connection telemetry stuff.